wiki/Game_mode/osu!/en.md

```diff
@@ -56,29 +34,27 @@ The number on the bottom left is the combo counter/score multiplier.
 
 ![Hit circles](/wiki/shared/osu_hitcircles.jpg "osu! hit circles")
 
-A thin, similarly-coloured approach circle on the outside of the hit circle will shrink over time. Tap on the note at the exact point when it touches the white border of the hit circle. A score burst will appear after tapping the hit circle, showing the score and health changes depending on timing of the tap to the beat. The hit circle will disappear after the score burst.
+Coloured circles with numbers on top of them, called hit circles, will appear on the playfield when playing. A thin, similarly-coloured [approach circle](/wiki/Gameplay/Hit_object/Approach_circle) on the outside of the hit circle will shrink over time. Tap on the hit circle at the exact point when the approach circle touches its white border. Note that you need to tap these hit circles in the order indicated by the number.
 
-To understand the hidden charm of hit circles, do not rely on perception only; feel the beatmap's beats and rhythm. It will help in juggling some of the visually-complex hit circles patterns found in later difficulties.
+A score burst will appear after tapping the hit circle, showing the score and health changes depending on timing of the tap to the beat. The hit circle will disappear after the score burst.
 
 #### Sliders
 
 ![Sliders](/wiki/shared/osu_slider.jpg "osu! sliders")
 
-First, tap on it at the right moment, however, it does not end there. A ball will begin to move across the path. Hold the mouse/keyboard-button (or keep the pen onto the tablet) and follow the ball within the thick orange circle, surrounding the ball, as it moves. The orange outer circle, called the follow circle, will appear if when holding onto the slider's ball, but will disappear when released.
+First, tap on the circle at the beginning of the slider (also known as the [sliderhead](/wiki/Gameplay/Hit_object/Slider/Sliderhead)) at the right moment. Once you tap on it, a ball will begin to move across the path. The orange outer circle, called the follow circle, will appear when holding onto the slider's ball, but will disappear when the cursor is outside the circle or the button is released. Hold the mouse/keyboard button (or keep the pen onto the tablet) and follow the ball within the follow circle as it moves.
```

Comment:
i suppose this hasn't been propagated throughout the wiki but "slider head" is preferred instead of "sliderhead"

```suggestion
First, tap on the circle at the beginning of the slider, called the [slider head](/wiki/Gameplay/Hit_object/Slider/Sliderhead), at the right moment. When tapped, a ball will begin to move across the path. The orange outer circle, called the follow circle, will appear when holding onto the slider's ball, but will disappear when the cursor is outside the circle or the button is released. Hold the mouse/keyboard button (or keep the pen on the tablet) and follow the ball within the follow circle as it moves.
```